SpywareFilter/sections/allowlist.txt

```diff
@@ -37,6 +37,9 @@
 !#endif
 !
 !
+! 
+! https://github.com/AdguardTeam/AdguardFilters/pull/141838
+@@||stat.tiara.kakao.com/track?d=*%22%3A%22go_transit%22%$domain=map.kakao.com
```

Comment:
I've been checking this for some time, but it is not enough on my end.
This rule helps. Unfortunately, the redirect one does not, otherwise I cannot see the bus route. @piquark6046 please also check it.
```adblock
@@||stat.tiara.kakao.com/track?$domain=map.kakao.com
```

Reply:
The rule suggested by you can help on my side too.
But, I think that allowing the tracking request has to be strict.
Can you provide me the related requests via Slack DM?

Anyway, a `$redirect` rule does not help on my side too.

Reply:
Or, please try the following rule:
```adblock
@@||stat.tiara.kakao.com/track?d=*%22go_transit%22%$domain=map.kakao.com

```

Reply:
Yep, as we discussed :)